src/main/java/org/jenkinsci/plugins/credentialsbinding/impl/StringBinding.java

```diff
@@ -50,6 +51,7 @@
         return new SingleEnvironment(getCredentials(build).getSecret().getPlainText());
     }
 
+    @Symbol("string")
```

Comment:
I had seen that already, yes -- I'll downgrade it an 🐜  because collisions aren't as bad as I thought, but think it's going to be immortalized in pipeline scripts better to be a little more descriptive.